scale/docs/rest/node.rst

```diff
@@ -337,75 +336,142 @@ These services provide access to information about the nodes.
+|               "disk_out_scheduled": 0.0,                                                                                |
+|               "disk_total_scheduled": 0.0                                                                               |
+|           }                                                                                                             |
+|       ],                                                                                                                |
+|       "resources": {                                                                                                    |
+|           "total": {                                                                                                    |
+|               "cpus": 16.0,                                                                                             |
+|               "mem": 63305.0,                                                                                           |
+|               "disk": 131485.0                                                                                          |
+|           },                                                                                                            |
+|           "scheduled": {                                                                                                |
+|               "cpus": 12.0,                                                                                             |
+|               "mem": 35392.0,                                                                                           |
+|               "disk": 131408.0                                                                                          |
+|           },                                                                                                            |
+|           "used": {                                                                                                     |
+|               "cpus": 16.0,                                                                                             |
+|               "mem": 63305.0,                                                                                           |
+|               "disk": 131485.0                                                                                          |
+|           }                                                                                                             |
+|       }                                                                                                                 |
+|   }                                                                                                                     |
 +--------------------+-------------------+--------------------------------------------------------------------------------+
 | **Error Responses**                                                                                                     |
```

Comment:
Remove this "Error Responses" section. We don't use this anywhere else in the REST docs.